### PR TITLE
Update Cozy-Vcard

### PR DESCRIPTION
**Pull request by clochix**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,10 @@
   "scripts": {
     "start": "node build/server.js",
     "test": "cake --use-js tests",
-    "build": "cake build"
+    "build": "cake build",
+    "lint": "npm run lint1 || npm run lint2",
```
Comment (aenario) [inline package.json]: mmh, if you use the || instead of the &&, wont it prevent using the command in scripts ? 

(like npm run build && npm run publish)

Comment (clochix) [inline package.json]: Yes. I use `||` so the client lint will run even if there are errors on server lint.
Depend if we want error in linting prevent next step (for example in build), or only use this command for informative purpose.

**Comment by clochix** [comment]
I will update this PR with the update of the JSDAV dependency once it has been published.

**Comment by aenario** [comment]
Update of jsdav included in branch developement (there was an issue and I had to republish it)

**Comment by clochix** [comment]
Thanks @aenario ☺

**Pull request merged**